platform/ios/src/MGLMapViewDelegate.h

```diff
@@ -277,6 +277,17 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (nullable UIImage *)mapView:(MGLMapView *)mapView didFailToLoadImage:(NSString *)imageName;
 
+/**
```

Comment:
This method appears under the “Managing Callout Views” heading in the header and in documentation:

https://github.com/mapbox/mapbox-gl-native/blob/12cb77e5f22a05d66ad7fe30fd0bf62d50948214/platform/ios/src/MGLMapViewDelegate.h#L598

Create a new heading for “Managing the Style” or move this method up to the “Loading the Map” section.

Reply:
On iOS is under the right header.